delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/identities/IdentitiesImpl.scala

```diff
@@ -133,10 +142,14 @@ object IdentitiesImpl {
     *   the cache configuration
     */
   def apply(realms: Realms, hc: HttpClient, config: CacheConfig): IO[Identities] = {
+    val groupsCache = LocalCache[String, Set[Group]](config)
+    val realmCache  = LocalCache[String, Option[Realm]](config)
```

Comment:
Not necessarily for this change but [mules](https://github.com/davenverse/mules/) is a nice alternative to caffeine cache (cats effect friendly and optimised)

Reply:
There is a caffeine dependency so it looks like it is using it behind the scene

Reply:
Oh yeah, I think that must be if you just want a functional wrapper but no change in behaviour (maybe that is preferable for us?). [`MemoryCache`](https://github.com/davenverse/mules/blob/main/modules/core/src/main/scala/io/chrisdavenport/mules/MemoryCache.scala#L311-L323) does the whole thing using CE (`MapRef` in the implementation)